src/gui/shellwidget/shellwidget.cpp

```diff
@@ -4,60 +4,62 @@
 #include "shellwidget.h"
 #include "helpers.h"
 
-ShellWidget::ShellWidget(QWidget *parent)
-:QWidget(parent), m_contents(0,0), m_bgColor(Qt::white),
-	m_fgColor(Qt::black), m_spColor(QColor()), m_lineSpace(0)
+ShellWidget::ShellWidget(QWidget* parent)
+: QWidget(parent), m_contents(0, 0), m_bgColor(Qt::white), m_fgColor(Qt::black),
```

Comment:
SpaceBeforeCtorInitializerColon: false

Reply:
I had this one already, I think that no formatting option yet allows no space there, I'll try to look further